ci/travis/ci.sh

```diff
@@ -193,6 +194,23 @@ test_python() {
   fi
 }
 
+# For running large Python tests on Linux and MacOS.
+test_large() {
+  bazel test --config=ci "$(./scripts/bazel_export_options)" --test_env=CONDA_EXE --test_env=CONDA_PYTHON_EXE \
```

Comment:
You still need to run bazel-sharding.py to list and split the targets in here. 

Also you don't have to put them here. Just write them in yaml works too.

Reply:
This uses per-shard tag filter, why is `bazel-sharding.py` still needed? Putting the command here reduces duplication a little bit, but I'm open to move the bazel command back to yaml.

Reply:
I agree in future, probably we can convert all Linux Python tests to use the `bazel-sharding.py` script. But it will be a larger change.

Reply:
Ah i missed the `--test_tag_filters="large_size_python_tests_shard_${BUILDKITE_PARALLEL_JOB}"`. Thanks for explaining!